### PR TITLE
Mention RESTEasy Reactive in the Qute guide

### LINKED ISSUE #14550: Can't use Qute in reactive application
When trying to build an application using RestEasy Reactive and Qute for rendering responses from templates, this exception is raised:

```
java.lang.IllegalStateException: The 'quarkus-resteasy-reactive' and 'quarkus-resteasy' extensions cannot be used at the same time.
	at io.quarkus.resteasy.reactive.server.deployment.ResteasyReactiveProcessor.setupEndpoints(ResteasyReactiveProcessor.java:235)
```

Given Qute supports async rendering, I'd expect it to be useable in a Reactive application. Specifically, I'd like to use Qute to render server-sent events.

### PR DESCRIPTION
**Pull request by geoand**
Fixes: #14550

### CODE REVIEW

File: docs/src/main/asciidoc/qute-reference.adoc [hunk 1]
```diff
@@ -1446,8 +1446,8 @@ class Item {
 [[resteasy_integration]]
 === RESTEasy Integration
 
-If you want to use Qute in your JAX-RS application, you'll need to add the `quarkus-resteasy-qute` extension first.
-In your `pom.xml` file, add:
+If you want to use Qute in your JAX-RS application, then depending on which JAX-RS stack you are using, you'll need to register the proper extension first.
+If you are using RESTEasy Classic via the `quarkus-resteasy` extension, then in your `pom.xml` file, add:
```
Comment (gsmet) [inline docs/src/main/asciidoc/qute-reference.adoc:1450]:
```suggestion
If you are using the traditional `quakus-resteasy` extension, then in your `pom.xml` file, add:
```
maybe?

I'm not sure to be honest. I'm just wondering if marketing it as `RESTEasy Classic` vs `RESTEasy Reactive` is a good thing.

Comment (geoand) [reply docs/src/main/asciidoc/qute-reference.adoc:1450]: I don't know either. I've been using that terminology myself but that certainly doesn't mean it's the best

Comment (geoand) [reply docs/src/main/asciidoc/qute-reference.adoc:1450]: But let's go with your suggestion for now, as it doesn't commit us to anything whereas mine pretty much does

Comment (mkouba) [reply docs/src/main/asciidoc/qute-reference.adoc]: "traditional `quakus-resteasy`" sounds OK.

Comment (geoand) [reply docs/src/main/asciidoc/qute-reference.adoc]: OK, then the PR now is should be good :)

**Review by cescoffier — APPROVED** [review]

**Pull request merged**